build.ps1

```diff
@@ -117,7 +117,7 @@ if (!$SkipBuild) {
     New-Item -ItemType Directory $target > $null
 
 # make sure dependencies are built first so clippy runs correctly
-$windows_projects = @("pal", "ntreg", "ntstatuserror", "ntuserinfo", "registry", "reboot_pending", "wmi-adapter")
```

Comment:
Why "ntstatuserror" and "ntuserinfo" are removed?

Reply:
Those libs were used by the old registry resource for handling NT API errors (translating to human readable errors) and for getting the current user for HKCU (it's not a real hive).  None of these are needed as the new crate uses win32 apis that abstract all of this.